src/services/slack/slack.ts

```diff
@@ -33,23 +36,21 @@ const postMessage = async (channel: string, text: string) => {
 
 const postInteractivePrompt = async (
   companyName: string,
-  channel: string | undefined,
+  channel: string,
   blocks: KnownBlock[]
 ) => {
   const text = `Help me find a profile for ${companyName}.`;
-  if (channel) {
-    try {
-      await web.chat.postMessage({
-        channel: channel,
-        text: text,
-        blocks: blocks,
-      });
-    } catch (error) {
-      console.error(error);
-      return false;
-    }
-    return true;
-  } else return false;
+  try {
```

Comment:
should we check if channel is empty string? Does web.chat.postMessage handle that? 😄